src/jump.c

```diff
@@ -238,10 +234,8 @@ edit_jumpdb(void)
 	return EXIT_SUCCESS;
 }
 
-int
-dirjump(char **args)
 /* Jump into best ranked directory matched by ARGS */
-{
+int dirjump(char **args) {
```

Comment:
I write "int\ndirjump" instead of "int dirjump" just to make the function more easily searchable, that is, independently of the return type.

Reply:
good idea

Reply:
also you should put the function explanation outside of the function. I use folding in vim and the comments get trapped inside the fold. prob will happed with other text editors.